src/assets/css/inputs.css

```diff
@@ -659,35 +659,43 @@ input:checked + .input-slider:before {
     cursor: pointer;
     min-width: 200px;
     padding: .125rem .5rem;
+
+    &&:focus {
+        outline-width: 0;
+    }
+
+    &&.select-dropdown {
+        margin-top: .3125rem;
+        margin-bottom: .125rem;
+        border-bottom: none;
+
+        &&.input-mandatory {
+            border: 2px solid $brand-mandatory-color;
+        }
+
+        &&.focused {
+            border-bottom: 1px solid #4b5b64;
+        }
+
+        &&.opened {
+            border-bottom: none;
+        }
+    }
```

Comment:
Nice refactoring! 

Small heads-up: This will clash with https://github.com/metasfresh/metasfresh-webui-frontend/pull/1687/commits/2dee0652ea1336da44667e894f62d9d6ad50336c #1687